errors.go

```diff
@@ -77,7 +81,9 @@ func HandleExitCoder(err error) {
 	}
 
 	if exitErr, ok := err.(ExitCoder); ok {
-		if err.Error() != "" {
+		if _, ok := exitErr.(ErrorFormatter); ok {
```

Comment:
Ah, I see, it's just to avoid printing an empty line if there is no error message. Probably an edge case, but I could see someone setting an empty `ExitCoder` just to get the exit behavior without outputting a message.